### DISCUSSION TITLE
[form]Warnings in browser console during form validation

### DISCUSSION THREAD

**Discussion by vatosarmat** [element-plus/element-plus#11177] [Q&A] [answered]
Is possible to disable them?
![debugWarn](https://user-images.githubusercontent.com/35213999/210963060-b0a4ac59-83ef-416c-b295-424b9bdc5eea.png)

**Comment by chenxch** [comment, marked as answer]
This only occurs in non-production mode

**Reply by vatosarmat** [reply]
Maybe there should be a way to disable them. They could be annoying

**Reply by phoenix-yyj** [reply]
is there any plan to support disable this?

**Reply by chenxch** [reply]
There are currently no plans to add new configuration items. If you really don’t want to see them, you can modify the debugWarn method in node_modules/element-plus/(es/lib)/utils/error.js.

**Reply by lcastillo0136** [reply]
console.warn = () => {}

**Comment by azatgt1996** [comment]
Has anyone solved it?
